### PR TITLE
quotes fix

### PR DESCRIPTION
**Pull request by Andrsi13**

### CODE REVIEW

**Comment by svilav** [comment]
Oh, it seems like I was confused. Actually, the previous code is correct (we need to have different quote types when nesting). So it seems like you can close this PR, sorry for that.

Upd. I figured it out. The functions you changed were fine. The changes are needed in `get_today_birthday`

**Comment by svilav** [comment]
@TetianaFilonenko can you look plz, because I don't trust myself anymore :)

File: console_bot/birthdays_per_week.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def get_birthdays_per_week(users):
             update_result(result, user_name, birthday_this_year)
     if not result:
         return "No birthdays next week"
-    return "\n".join([f"{date}: {(',').join(people)}" for date, people in result.items()])
```
Comment (svilav) [inline console_bot/birthdays_per_week.py]: this was correct before

File: console_bot/birthdays_per_week.py [hunk 2]
```diff
@@ -34,7 +34,7 @@ def get_birthdays_in_next_days(users, days):
             result_dates(result, user_name, birthday_this_year)
     if not result:
         return f"No birthdays in next {days} days"
-    return "\n".join([f"{date}: {(',').join(people)}" for date, people in result.items()])
```
Comment (svilav) [inline console_bot/birthdays_per_week.py]: and this

File: console_bot/birthdays_per_week.py [hunk 3]
```diff
@@ -34,7 +34,7 @@ def get_birthdays_in_next_days(users, days):
...
+    return "\n".join([f"{date}: {(",").join(people)}" for date, people in result.items()])
 
 def get_today_birthday(users):
```
Comment (svilav) [inline console_bot/birthdays_per_week.py]: the changes are needed in return statement of this function

Comment (Andrsi13) [reply console_bot/birthdays_per_week.py]: Now should be good

**Review by TetianaFilonenko — COMMENTED** [review]
still working not as expected on console_bot/birthdays_per_week.py:50 
```
return '\n'.join([f'Don`t forget to congratulate: {(',').join(people)}' for people in result.values()])
```
Works with this  
```
return '\n'.join([f"Don't forget to congratulate: {', '.join(people)}" for people in result.values()])
```

**Review by TetianaFilonenko — APPROVED** [review]
Looks great!

**Pull request merged**